### PR TITLE
Feature: Countries crud pages

### PR DESCRIPTION
**Pull request by edbrett**
- Created controllers, models, and views for countries resource
- Add db migration with the fields :name, :iso, :shp_url, :geojson_url

### CODE REVIEW

File: app/controllers/admin/countries_controller.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+class Admin::CountriesController < ApplicationController
+  before_action :set_country, only: [:show, :edit, :update, :destroy]
+
+  # GET /countries
+  # GET /countries.json
+  def index
+    @countries = Country.all
```
Comment (simaob) [inline app/controllers/admin/countries_controller.rb]: let's order the countries by name. It's a good default=)

File: app/controllers/countries_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class CountriesController < ApplicationController
```
Comment (simaob) [inline app/controllers/countries_controller.rb]: No need for a CountriesController. As they will be used on a downloads controller.

File: app/controllers/admin/countries_controller.rb (hunk 1 above)
Comment (edbrett) [reply app/controllers/admin/countries_controller.rb]: agreed.

File: app/controllers/admin/countries_controller.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@ class Admin::CountriesController < AdminController
   # GET /countries
   # GET /countries.json
   def index
-    @countries = Country.all
+    @countries = Country.all.order(:name)
```
Comment (simaob) [inline app/controllers/admin/countries_controller.rb]: No need for "all" in here. Just use `Country.order(:name)`

More about it here: http://guides.rubyonrails.org/active_record_querying.html#ordering

**Pull request merged**